modules/op-release-notes-1-8.adoc

```diff
@@ -533,3 +533,69 @@ securityContext:
 * Before this update, pods for the prune cronjobs were not scheduled on infrastructure nodes, as expected. Instead, they were scheduled on worker nodes or not scheduled at all. With this update, these types of pods can now be scheduled on infrastructure nodes if configured in the `TektonConfig` custom resource (CR).
 // link:https://issues.redhat.com/browse/SRVKP-1806
 // Rupali Behera
+
+
+[id="release-notes-1-8-1_{context}"]
+== Release notes for {pipelines-title} General Availability 1.8.1
+
+With this update, {pipelines-title} General Availability (GA) 1.8.1 is available on {product-title} 4.10, 4.11, and the upcoming version.
+
+[id="known-issues-1-8-1_{context}"]
+=== Known issues
+
+* When installer sets associated with cluster tasks are in failed state, the status of the `TektonConfig` custom resource is incorrectly displayed as `True` instead of `False`.
```

Comment:
@sm43 Is this only related with clustertasks related installerset or with any installersets in general?

Reply:
in general, if there is any error in addon due to some reason
it was not getting reflected in tektonconfig
but this will be part of 1.9 not 1.8.1

Reply:
for 1.8.1 the installer set was failing due to missing clustertask field, that bug is fixed.

Reply:
This is part of known issues. @sounix000 this is not only related to clustertask related installersets, if any installerset is in error state the config is shown as `True`